PWGDQ/Core/VarManager.h

```diff
@@ -750,6 +751,9 @@ void VarManager::FillTrack(T const& track, float* values)
         if (fgUsedVars[kTrackDCAresZ]) {
           values[kTrackDCAresZ] = std::sqrt(track.cZZ());
         }
+        if (fgUsedVars[kPt_in]) {
+          values[kPt_in] = values[kPin] / std::sqrt(1.0 + track.tgl() * track.tgl());
+        }
```

Comment:
Two comments.
1) The kPt_IN is not properly computed here, since tgl does not correspond to the track parameter at the inner TPC wall. Why is this needed?
2) Why is this needed here ? This code block is for the TrackDCA, so the kPt_in variable should be computed below, at L794, where you have it a second time.